highfidelity/hifi
# MS16192: Fix Near Grabbing After Using Web Entity

- Fix for MS[#16192](https://highfidelity.manuscript.com/f/cases/16192/Cannot-near-grab-when-hand-is-pointed-at-a-web-entity-or-tablet-screen)

**TEST PLAN**
- Launch Interface
- Enter VR mode
- Create/locate a web entity or locate the tablet screen
- Create/locate a grabbable object or locate the tablet
- Point your hand at the web entity or tablet screen so that a blue laser appears (do not use trigger)
- With the blue laser still on the screen, attempt to near grab the grabbable object or tablet
- Near grabbing should work
